packages/react/src/components/DetailsList/DetailsRowCheck.types.ts

```diff
@@ -36,6 +37,11 @@ export interface IDetailsRowCheckProps extends React.HTMLAttributes<HTMLElement>
    */
   canSelect: boolean;
 
+  /**
+   * Selection mode
+   */
+  selectionMode?: SelectionMode | undefined;
```

Comment:
I think you don't need the `undefined` here since you are already using `?` at the end of the prop name.

Reply:
fixed!